### DISCUSSION TITLE
Task 5 - Assignment 7 - Question

### DISCUSSION THREAD

**Discussion by derekscholes** [USU-ITLS-x265-23-summer/class-repo#3] [Week 1 - Questions] [answered]
This assignment is a statement and doesn't supply any question or request. Is there anything that we need to do to fulfill this requirement, or do you want us to only review the materials you have provided in the "references" folder?

**Comment by CraigButikofer** [comment, marked as answer]
@derekscholes I think that Task 5 Assignment 7 is a direction/guidance setting up the overall expectation to build out the repository. I read it like the general expectation is to build out the collective knowledge repo as we go along, but this first week is a little awkward because it is difficult for us to talk about something so new in real time because its so new. So rather I see this "assignment" as more of a reminder to cycle back and make sure that we are actually building out that repository and helping to to become a helpful resource for us and our peers. I could be wrong though.

**Reply by beto0890** [reply]
Thank you so much for providing the information!

**Comment by metasean** [comment]
@CraigButikofer - That's an awesome answer!

Yes, most weeks the expectation is that you will each add notes, examples, clarifications, etc. as you're working through the various exercises.  However, to do that you have to understand _how_ to do it, and learning how to do that is the primary goal of this week.

Since you have undoubtedly learned _something_ this week, Assignment 7 is to actually use what you've learned this week and add some bit of that new knowledge to the class-repo.  This could be something as simple as adding a new term and definition to the glossary page or adding a new page summarizing the different steps in the lifecycle of code to be added to an existing repo, or just a page of markdown examples.
